README.md

```diff
@@ -51,6 +51,22 @@ Your `DATABASE_URL` should look like `mysql://<User>:<Password>@<Host>:4000/book
 
 </details>
 
+## Run Development Server
+
+```bash
+# install dependencies
+yarn
+# config database URL
+export DATABASE_URL='mysql://<User>:<Password>@<Host>:4000/bookshop?pool_timeout=60&sslaccept=accept_invalid_certs'
```

Comment:
TLS parameter will auto-add to DATABASE_URL. So we can just set as`mysql://<User>:<Password>@<Host>:4000/bookshop`.